Test Failure: style applies tranform as transition on remove FAILED

I noticed below test failure upon running `npm test` on my machine. A PR is coming up soon.
```
> karma start

05 09 2019 18:19:31.761:INFO [compiler.karma-typescript]: Compiling project using Typescript 3.3.4000
05 09 2019 18:19:36.427:INFO [compiler.karma-typescript]: Compiled 26 files in 4606 ms.
05 09 2019 18:19:37.335:INFO [bundler.karma-typescript]: Bundled imports for 26 file(s) in 404 ms.
05 09 2019 18:19:38.801:INFO [karma-server]: Karma v3.1.4 server started at http://0.0.0.0:9876/
05 09 2019 18:19:38.803:INFO [launcher]: Launching browsers Chrome, Firefox with concurrency unlimited
05 09 2019 18:19:38.811:INFO [launcher]: Starting browser Chrome
05 09 2019 18:19:38.829:INFO [launcher]: Starting browser Firefox
05 09 2019 18:19:40.721:INFO [Chrome 76.0.3809 (Windows 10.0.0)]: Connected on socket a2jgjAhxrFJuEPpKAAAA with id 27571023
................................................................................
...................................................................
Chrome 76.0.3809 (Windows 10.0.0): Executed 147 of 147 SUCCESS (1.075 secs / 0.862 secs)
05 09 2019 18:19:44.824:INFO [Firefox 69.0.0 (Windows 10.0.0)]: Connected on socket bzlBMsLmXtLqFFD7AAAB with id 22765761
................................................................................
.....................................................
Firefox 69.0.0 (Windows 10.0.0) style applies tranform as transition on remove FAILED
        uncaught exception: AssertionError: 1 === 0 (:0)
.............
Firefox 69.0.0 (Windows 10.0.0): Executed 147 of 147 (1 FAILED) (0.911 secs / NaN secs)
TOTAL: 1 FAILED, 293 SUCCESS
npm ERR! Test failed.  See above for more details.
```

--- a/test/style.js
+++ b/test/style.js
@@ -118,7 +118,7 @@ describe('style', function() {
   });
   it('applies tranform as transition on remove', function(done) {
     var btn = h('button', { style: {
-        transition: 'transform 0.5s',
+        transition: 'transform 0.1s',
         remove: { transform: 'translateY(100%)' }
     }}, ['A button']);
     var vnode1 = h('div.parent', {}, [btn]);
@@ -126,11 +126,12 @@ describe('style', function() {
     document.body.appendChild(vnode0);
     patch(vnode0, vnode1);
     patch(vnode1, vnode2);
-    assert.strictEqual(document.querySelectorAll('button').length, 1);
-    setTimeout(function () {
-        assert.strictEqual(document.querySelectorAll('button').length, 0);
-        done();
-    }, 700);
+    const button = document.querySelector('button');
+    assert.notStrictEqual(button, null);
+    button.addEventListener('transitionend', function() {
+      assert.strictEqual(document.querySelector('button'), null);
+      done();
+    });
   });
   describe('using toVNode()', function () {
     it('handles (ignoring) comment nodes', function() {

